scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -96,6 +97,7 @@ abstract class ScanResultsStorage {
                 is FileBasedStorageConfiguration -> createFileBasedStorage(config)
                 is PostgresStorageConfiguration -> createPostgresStorage(config)
                 is ClearlyDefinedStorageConfiguration -> createClearlyDefinedStorage(config)
+                is Sw360StorageConfiguration -> TODO()
```

Comment:
Actually, even if we do not have the scan storage implementation yet, I believe we need to still add support for reading in the config here; otherwise this implementation cannot be used / tested as-is.

Reply:
I don't really get it.. How can i add support in `ScanResultStorage` if the SW360 storage is not implemented yet?

Reply:
What I mean is: This PR already could (and should, IMO) have support for reading the SW360 storage *config*, as it is being (slightly mis-)used here, but this PR should not yet implement the actual SW360 storage backend.

Reply:
I find it a bit strange to configure the SW360 connection in the storage section and then use this from multiple different components. Should there be something like a central configuration for external services? (This is of course not in the scope of this PR.)

Reply:
@sschuberth There is already support for reading the configuration with my changes in this PR. The implemented `SW360StorageConfiguration` makes it possible to read the config file. 

@oheger-bosch Due to the experimental state, it is read this way. But of course, there should be a central configuration for that.